src/components/Breadcrumbs/__tests__/__snapshots__/BreadcrumbItem.spec.js.snap

```diff
@@ -58,34 +62,38 @@ exports[`<Breadcrumb.Item /> renders correctly when no href prop is passed 1`] =
   cursor: pointer;
 }
 
-.c2:focus,
-.c2:active,
-.c2:visited,
-.c2:hover {
+.c1:focus,
+.c1:active,
+.c1:visited,
+.c1:hover {
   color: rgba(255,255,255,1);
 }
 
-.c2:hover {
+.c1:hover {
   color: #0150a7;
 }
 
 .c0 {
   white-space: nowrap;
 }
 
-.c1 {
+.c2 {
   cursor: text;
 }
 
-.c1:hover {
+.c2:hover {
   color: currentColor;
 }
 
+@media screen and (min-width:480px) {
+
+}
+
 <li
   className="c0"
 >
   <span
-    className="c1 c2"
+    className="sc-AxhUy sc-AxgMl c1 c2"
```

Comment:
My understanding is that there should not be any generated classnames in the snapshots, since we are using the jest-styled-components serializer. Could you please investigate this change, and post your findings here? Thanks!

Reply:
There opened issues in jest-styled-components related to the described above problems with generated class names. It seems like jest-styled-components can't handle in proper way nested styled-components.
https://github.com/styled-components/jest-styled-components/issues/276
https://github.com/styled-components/jest-styled-components/issues/307